src/xdsl/xdsl_opt_main.py

```diff
@@ -221,8 +255,8 @@ def setup_pipeline(self):
 
     def parse_input(self) -> ModuleOp:
         """
-        Parse the input file by invoking the parser specified by the `parser` 
```

Comment:
Could we remove those changes from this PR? This makes the blame quite unreadable.